app/assets/stylesheets/modules/master_overlay.css.scss

```diff
@@ -418,13 +418,14 @@ $parent-width: $parent-split * 0.21;
 }
 
 .row {
+  -js-display: flex;
```

Comment:
I thought you had said -js-display messed up IE 11. Is it only if flexibility is loaded?